Azure/azure-cli
# [AppService] functionapp: AzureWebJobsDashboard will only be set if AppInsights is disabled

Resolves #13236 

**Description**  

This PR changes `az functionapp create` so that it only sets the `AzureWebJobsDashboard` app setting if AppInsights is disabled. If AppInsights is enabled, `AzureWebJobsDashboard` is ignored so it's not necessary and shouldn't be set. If AppInsights is disabled, then the dashboard is the only way the customer would be able to get logs.

**Testing Guide**  

Changed two tests to make sure `AzureWebJobsDashboard` is set in the correct circumstances. Running `azdev test FunctionAppWithAppInsightsDefault` tests it.

**History Notes**  


[AppService] az functionapp create: AzureWebJobsDashboard will only be set if AppInsights is disabled

---

This checklist is used to make sure that common guidelines for a pull request are followed.
